optimize/backend/src/it/java/io/camunda/optimize/service/importing/ZeebeUserTaskImportIT.java

```diff
@@ -879,6 +880,31 @@ public void doNotImportCandidateGroupUpdates() {
         .containsOnly(Collections.emptyList());
   }
 
+  @Test
+  public void importOtherDueDateFormat() {
+    // given
+    final ProcessInstanceEvent instance =
+        deployAndStartInstanceForProcess(
+            createSimpleNativeUserTaskProcess(TEST_PROCESS, "2023-03-02T15:35+02:00"));
```

Comment:
This is a different format from the one we're actually expecting from zeebe and its ony used in this one test, but I ll add it as a var in the test case since its used twice